lib/vagrant-service-manager/command.rb

```diff
@@ -266,21 +265,17 @@ def print_vagrant_box_version(script_readable = false)
       end
 
       def restart_service(service)
-
-        if SCCLI_SERVICES.include? service
-          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
-          command = "sudo sccli #{service}"
-        else
-          command = "sudo systemctl restart #{service}"
-        end
-
-        with_target_vms(nil, { single_target: true}) do |machine|
-          machine.communicate.execute(command) do |type, error|
-            if type == :stderr
-              @env.ui.error(error)
-              exit 126
-            end
+        errors = []
+        command = "systemctl restart #{service}"
```

Comment:
I believe we should really be relying on sccli to contain all of the knowledge about a service.  If we want to be using `systemctl restart` then we should have sccli smart enough to do that.  Perhaps we should just extend sccli to accept "start" and "restart" as commands.  The complexity of building the interface seems easier to maintain than splitting the application lifecycle management across multiple programs and repositories.

Regarding "k8s" I believe we should drop support for this term.  It isn't a system term and it isn't helping us to have to do adhoc translation.  If we don't drop it, then it should be up to sccli to do the right thing .. not vagrant-service-manager.